lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,23 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+    token_list = []
```

Comment:
try to think about how to use a string method instead of using a list

Reply:
changed to str, using replace and split